owncloudData/src/test/java/com/owncloud/android/data/user/datasources/OCRemoteUserDataSourceTest.kt

```diff
@@ -86,10 +87,12 @@ class OCRemoteUserDataSourceTest {
 
         assertNotNull(userInfo)
         assertEquals(OC_USER_INFO, userInfo)
+
+        verify(exactly = 1) { ocUserService.getUserInfo() }
     }
 
     @Test(expected = Exception::class)
-    fun getUserInfoException() {
+    fun `getUserInfo returns an exception when service receive a exception`() {
```

Comment:
```suggestion
    fun `getUserInfo throws an exception when service receive a exception`() {
```
But just as we discussed, these tests could be omitted